config/k8s/end-to-end.yaml

```diff
@@ -0,0 +1,96 @@
+# A basic flag custom resource
+apiVersion: core.openfeature.dev/v1alpha1
+kind: FeatureFlagConfiguration
+metadata:
+  name: end-to-end
+  labels:
+    app: open-feature-demo
+spec:
+  featureFlagSpec: |
+    {
+      "flags": {
+        "new-welcome-message": {
+          "state": "ENABLED",
+          "variants": {
+            "on": true,
+            "off": false
+          },
+          "defaultVariant": "on"
+        },
+        "hex-color": {
+          "variants": {
+            "red": "c05543",
+            "green": "2f5230",
+            "blue": "0d507b",
+            "yellow": "yellow"
+          },
+          "defaultVariant": "blue",
+          "state": "ENABLED"
+        },
+        "fib-algo": {
+          "variants": {
+            "recursive": "recursive",
+            "memo": "memo",
+            "loop": "loop",
+            "binet": "binet"
+          },
+          "defaultVariant": "recursive",
+          "state": "ENABLED",
+          "targeting": {
+            "if": [
+              {
+                "in": ["@faas.com", {
+                  "var": ["email"]
+                }]
+              }, "binet", null
+            ]
+          }
+        }
+      }
+    }
+---
+# Deployment of a demo-app using our custom resource
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: open-feature-demo-deployment
+  labels:
+    app: open-feature-demo
```

Comment:
Added this label everywhere to make it possible to `kubectl wait` the application of this whole file.